lang/test/org/partiql/lang/eval/EvaluatingCompilerGroupByTest.kt

```diff
@@ -103,6 +102,16 @@ class EvaluatingCompilerGroupByTest : EvaluatorTestBase() {
             val compilationOptions: List<CompOptions> = CompOptions.values().toList()
         )
 
+        /**
+         * The [EvaluatorTestTarget.PLANNER_PIPELINE] does NOT support [UndefinedVariableBehavior.MISSING], so if the
+         * [compOptions] includes the [UndefinedVariableBehavior], we should use the [EvaluatorTestTarget.COMPILER_PIPELINE]
+         */
+        private fun getTestTarget(compOptions: CompOptions, default: EvaluatorTestTarget): EvaluatorTestTarget = when (compOptions) {
```

Comment:
See [the PartiQLCompilerPipelineFactory's note on this](https://github.com/partiql/partiql-lang-kotlin/blob/11c265e08f45abb7be7ebaa5f51fe58aa4bc0eb3/lang/test/org/partiql/lang/eval/evaluatortestframework/PartiQLCompilerPipelineFactory.kt#L39).